실험 'pythonaa'이(가) 활성 상태입니다.
실험 'pythonPromptNewFormatterExt'이(가) 활성 상태입니다.
실험 'pythonPromptNewToolsExt'이(가) 활성 상태입니다.
실험 'pythonSurveyNotification'이(가) 활성 상태입니다.
실험 'pythonTerminalEnvVarActivationcf'이(가) 활성 상태입니다.
실험 'pythonTestAdapter'이(가) 활성 상태입니다.
LSP Notebooks experiment is enabled
LSP Notebooks interactive window support is enabled
> conda info --json
Python 인터프리터 경로: /bin/python3
> /bin/python -I ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
[ERROR 2023-8-26 5:18:25.424]: Support for Python 2.7 has been dropped by the Python extension so certain features may not work, upgrade to using Python 3.
> /bin/python ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> /bin/python3 -I ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> /usr/bin/python -I ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
[ERROR 2023-8-26 5:18:28.435]: Support for Python 2.7 has been dropped by the Python extension so certain features may not work, upgrade to using Python 3.
> /usr/bin/python ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> /usr/bin/python3 -I ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> /usr/include/python3.7 -I ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
[ERROR 2023-8-26 5:18:29.87]: [Error: Command failed: /usr/include/python3.7 -I /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
/bin/sh: 1: /usr/include/python3.7: Permission denied

	at ChildProcess.exithandler (node:child_process:408:12)
	at ChildProcess.emit (node:events:526:28)
	at maybeClose (node:internal/child_process:1092:16)
	at Socket.<anonymous> (node:internal/child_process:451:11)
	at Socket.emit (node:events:526:28)
	at Pipe.<anonymous> (node:net:687:12)] {
  killed: false,
  code: 126,
  signal: null,
  cmd: '/usr/include/python3.7 -I /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py'
}
> ~/miniconda3/bin/python -I ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
Send text to terminal: /bin/python3 "/home/pi/Desktop/python camera/python_camera.py"
> /usr/include/python3.7 -I ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
[ERROR 2023-8-26 5:18:31.188]: [Error: Command failed: /usr/include/python3.7 -I /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
/bin/sh: 1: /usr/include/python3.7: Permission denied

	at ChildProcess.exithandler (node:child_process:408:12)
	at ChildProcess.emit (node:events:526:28)
	at maybeClose (node:internal/child_process:1092:16)
	at Socket.<anonymous> (node:internal/child_process:451:11)
	at Socket.emit (node:events:526:28)
	at Pipe.<anonymous> (node:net:687:12)] {
  killed: false,
  code: 126,
  signal: null,
  cmd: '/usr/include/python3.7 -I /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py'
}
> -I ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
[ERROR 2023-8-26 5:18:32.989]: [Error: Command failed: -I /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
/bin/sh: 0: Illegal option - 

	at ChildProcess.exithandler (node:child_process:408:12)
	at ChildProcess.emit (node:events:526:28)
	at maybeClose (node:internal/child_process:1092:16)
	at ChildProcess._handle.onexit (node:internal/child_process:302:5)] {
  killed: false,
  code: 2,
  signal: null,
  cmd: '-I /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py'
}
> -I ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ~/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
[ERROR 2023-8-26 5:18:35.213]: [Error: Command failed: -I /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
/bin/sh: 0: Illegal option - 

	at ChildProcess.exithandler (node:child_process:408:12)
	at ChildProcess.emit (node:events:526:28)
	at maybeClose (node:internal/child_process:1092:16)
	at Socket.<anonymous> (node:internal/child_process:451:11)
	at Socket.emit (node:events:526:28)
	at Pipe.<anonymous> (node:net:687:12)] {
  killed: false,
  code: 2,
  signal: null,
  cmd: '-I /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py /home/pi/.vscode/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py'
}
Pylance 언어 서버를 시작하는 중입니다.
